frontend/src/components/kl-char.js

```diff
@@ -0,0 +1,55 @@
+import store from './store.js';
+import { clone } from 'domain/utility';
+import { renderValidationErrors } from '../render';
+export default {
+    data: function () {
+        return {
+            _id: undefined,
+            char: {
+                atkSpd: 0,
+                dmg: 0,
+                critChance: 0,
+                critDmg: 0
+            },
+            store
```

Comment:
You should use the store in the same way as $http and $t. So in the main entry js just do
```js
import store from './store.js';
Vue.prototype.$store = store;
```
Or even better write a proper vue plugin for that https://vuejs.org/v2/guide/plugins.html
And then in the components you can use it like so `this.$store.setCharAction`...